packages/realm/rollup.config.mjs

```diff
@@ -16,17 +16,26 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
+/* eslint-env node */
+
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 import commonjs from "@rollup/plugin-commonjs";
 import json from "@rollup/plugin-json";
 import typescript from "@rollup/plugin-typescript";
 import replace from "@rollup/plugin-replace";
 import dts from "rollup-plugin-dts";
+import istanbul from "rollup-plugin-istanbul";
 
 import pkg from "./package.json" assert { type: "json" };
 
 const mainExport = pkg.exports["."];
 
+const { ENABLE_TEST_COVERAGE_INSTRUMENTATION } = process.env;
+if (ENABLE_TEST_COVERAGE_INSTRUMENTATION) {
```

Comment:
Introducing an environment variable `ENABLE_TEST_COVERAGE_INSTRUMENTATION` which injects the istanbul instrumentation into the SDK bundle.